packages/internal-test-helpers/lib/test-cases/abstract-application.ts

```diff
@@ -30,17 +39,24 @@ export default class AbstractApplicationTestCase extends AbstractTestCase {
     return instance;
   }
 
-  get element() {
+  _element: Element | null = null;
+
+  get element(): Element | null {
     if (this._element) {
       return this._element;
-    } else if (ENV._APPLICATION_TEMPLATE_WRAPPER) {
-      return (this._element = document.querySelector('#qunit-fixture > div.ember-view'));
+    }
+
+    let element;
+    if (ENV._APPLICATION_TEMPLATE_WRAPPER) {
+      element = document.querySelector('#qunit-fixture > div.ember-view');
     } else {
-      return (this._element = document.querySelector('#qunit-fixture'));
+      element = document.querySelector('#qunit-fixture');
     }
+
+    return (this._element = element);
```

Comment:
Absolutely not a blocker, since this code was already here, but I find this *way* clearer:

```ts
if (!this._element) {
  this._element = ENV._APPLICATION_TEMPLATE_WRAPPER
    ? document.querySelector('#qunit-fixture > div.ember-view')
    : document.querySelector('#qunit-fixture');
}

return this._element;
```